apps/docs/content/guides/realtime/authorization.mdx

```diff
@@ -0,0 +1,293 @@
+---
+id: 'authorization'
+title: 'Realtime Authorization'
+description: 'Authorization for Supabase Realtime'
+sidebar_label: 'Authorization'
+---
+
+You can control user access to Realtime Channels using RLS policies. By applying RLS, you can:
+
+- Control which users can broadcast to a Channel
+- Control which users can receive broadcasts from a Channel
+- Control which users can publish their presence to a Channel
+- Control which users can receive messages about the presence of other users
+
+<Admonition type="caution">
+
+Realtime Authorization is in public alpha. To use Authorization for your Realtime Channels, use `realtime.js` version `2.10.0` or later.
+
+</Admonition>
+
+## How it works
```

Comment:
I think we left out how it makes a transaction and rolls it back on `realtime.messages`

Reply:
Added.